vaccine_feed_ingest/runners/nc/myspot_gov/normalize.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest.schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+
+def _get_id(site: dict) -> str:
+    loc_id = site["Event Location Id"]
+    return f"nc:myspot_gov:{loc_id}"
+
+
+def _get_name(site: dict) -> str:
+    return site["Provider Location Name"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["Street Address"],
+        street2=site["Street Address 2"],
+        city=site["City"],
+        state=site["State"],
+        zip=site["Postal Code"],
+    )
+
+
+def _get_location(site: dict):
+    if site["latitude"] == "" or site["longitude"] == "":
+        return None
+    return schema.LatLng(
+        latitude=float(site["latitude"]),
+        longitude=float(site["longitude"]),
+    )
+
+
+def _get_contacts(site: dict):
+    return [
+        schema.Contact(
+            phone=site["Appointment Phone"],
+            website=site["Web Address"],
+        )
+    ]
+
+
+def _normalize_date(dt: str):
+    if dt == "":
+        return None
+    return dt[0:4] + "-" + dt[4:6] + "-" + dt[6:8]
+
+
+def _get_opening_dates(site: dict):
+    if site["Start Date"] == "" and site["End Date"]:
+        return None
+    return [
+        schema.OpenDate(
+            opens=_normalize_date(site["Start Date"]),
+            closes=_normalize_date(site["End Date"]),
+        )
+    ]
+
+
+def _get_inventories(site: dict):
+    ret = []
+    if site["Moderna"] == "Y":
+        ret.append(schema.Vaccine(vaccine="moderna", supply_level="in_stock"))
+    if site["Pfizer"] == "Y":
+        ret.append(schema.Vaccine(vaccine="pfizer_biontech", supply_level="in_stock"))
+    if site["Janssen"] == "Y":
+        ret.append(
+            schema.Vaccine(vaccine="johnson_johnson_janssen", supply_level="in_stock")
+        )
+    if site["Moderna"] == "N":
+        ret.append(schema.Vaccine(vaccine="moderna", supply_level="out_of_stock"))
+    if site["Pfizer"] == "N":
+        ret.append(
+            schema.Vaccine(vaccine="pfizer_biontech", supply_level="out_of_stock")
+        )
+    if site["Janssen"] == "N":
+        ret.append(
+            schema.Vaccine(
+                vaccine="johnson_johnson_janssen", supply_level="out_of_stock"
+            )
+        )
+    return ret
+
+
+def _get_organization(site: dict):
+    if site["Organization Name"] == "":
+        return None
+    if site["Organization Name"] == "Walmart, Inc.":
+        return schema.Organization(name=site["Organization Name"], id="walmart")
```

Comment:
Thank you for doing this! I have [added it to the wiki](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema#organization-enum)